src/pages/Inventory.page.js

```diff
@@ -64,4 +69,32 @@ export class InventoryPage extends BaseSwagLabPage {
                 throw new Error('Sort option is not correct');
         }
     }
+
+    // get all Products locators
+    async getAllProducts() {
+        const array = await this.inventoryItems.all();
+        return array;
+    }
+
+    // get N random products from Products array
+    getRandomProducts(allProducts, count) {
```

Comment:
I would move this function to helper module -> `utils/helpers.js`
You might use this function in different page objects